### DISCUSSION TITLE
CFWheels Version v2.3.0

### DISCUSSION THREAD

**Discussion by bpamiri** [wheels-dev/wheels#1128] [Announcements]
This is the official 2.3.0 release. There were no new enhancements or bug fixes introduced from release candidate 1 so the change log is essentially the same as v2.3.0.rc.1.

<hr /><em>This discussion was created from the release <a href='https://github.com/cfwheels/cfwheels/releases/tag/v2.3.0'>CFWheels Version v2.3.0</a>.</em>

**Comment by blastiko** [comment]
Hi bpamiri
Sorry but i don't understand new CFWheels 2.3.0 release structure...
With version 2.2.0, i download and unzip cfwheels-2.2.0.zip file and i develop based on this directory structure (like reported in docs).
Now, with v.2.3.0, what file i must download to obtain same directory structure?
Thank you.

**Comment by bpamiri** [comment]
The new release structure splits the core wheels directory from the rest of the framework template files. So you have CFWheels Core and CFWheels Base Template. You actually install the template file, and it in turn pulls in the core files via the box.json dependency settings.

So in command box you can issue this command:
install cfwheels-base-template

That pulls in the template files which is everything except the wheels folder.

Then issue the following command:
install

This will pull in the dependencies. 

Then you can start the server by issuing:
server start

If you use this structure then to upgrade to 2.4.0 when it is released, you would go into box.json and change the version from 2.3.0 to 2.4.0 and issue install again and the new version would be pulled in.

If you're more comfortable with the old style copying down a zip file, I've attached the old standalone zip file to the release:
https://github.com/cfwheels/cfwheels/releases/download/v2.3.0/cfwheels-2.3.0.zip

**Reply by blastiko** [reply]
Ok, now everything is clear to me.
And thank you so much for the old style zip too.
